apache/superset
# chore: Change copy to Edit chart in Dashboard dropdown

### SUMMARY
As shown in the screenshot

### BEFORE

<img width="759" alt="Clipboard 2022-21-04 at 1 28 26 PM" src="https://user-images.githubusercontent.com/60598000/167908267-3f003fb6-f99c-40e6-ada3-8ce63509dadb.png">

### AFTER

![Screen Shot 2022-05-11 at 19 13 28](https://user-images.githubusercontent.com/60598000/167908171-dc6097cc-d660-4078-9eea-ea198dff82fb.png)

### TESTING INSTRUCTIONS
1. Go to a Dashboard
2. Open the dropdown
3. Make sure the copy is right

### ADDITIONAL INFORMATION
